src/app/course/challenge/course-challenge-snippet/course-challenge-snippet.component.html

```diff
@@ -1,5 +1,19 @@
 <div class="flex flex-col gap-5">
     <ng-container *ngIf="course else skeletonTemplate">
+        <div layout="column"
```

Comment:
For the layout/spacing, we are using tailwind. You have to use tailwind classes. For example, you can use this https://tailwindcss.com/docs/margin for margin.